src/aws/nist-800-53-rev4/README.md

```diff
@@ -56,33 +56,71 @@ Policy Pack based on the [800-53 Rev. 4](https://csrc.nist.gov/publications/deta
-| AWS NIS 3.1  | CloudTrail log files should be encrypted with customer managed KMS keys                                  |
-| AWS NIS 3.2  | CloudWatch log groups should be encrypted with customer managed KMS keys                                 |
-| AWS NIS 3.3  | DynamoDB tables should be encrypted with AWS or customer managed KMS keys                                |
-| AWS NIS 3.4  | EBS volume encryption should be enabled                                                                  |
-| AWS NIS 3.5  | RDS instances should be encrypted                                                                        |
-| AWS NIS 3.6  | S3 bucket server-side encryption should be enabled                                                       |
-| AWS NIS 3.7  | SQS queue server-side encryption should be enabled with KMS keys                                         |
-| AWS NIS 4.1  | CloudFront distribution origin should be set to S3 or origin protocol policy should be set to https-only |
-| AWS NIS 4.2  | CloudFront viewer protocol policy should be set to https-only or redirect-to-https                       |
-| AWS NIS 4.3  | ElastiCache transport encryption should be enabled                                                       |
-| AWS NIS 4.4  | ELBv1 listener protocol should not be set to http                                                        |
-| AWS NIS 4.5  | S3 bucket policies should only allow requests that use HTTPS                                             |
-| AWS NIS 4.6  | SNS subscriptions should deny access via HTTP                                                            |
-| AWS NIS 6.1  | CloudFront access logging should be enabled                                                              |
-| AWS NIS 6.4  | CloudTrail should have at least one CloudTrail trail set to a multi-region trail                         |
-| AWS NIS 6.6  | CloudTrail trails should be configured to log management events                                          |
-| AWS NIS 6.8  | Exactly one CloudTrail trail should monitor global services                                              |
-| AWS NIS 6.9  | Load balancer access logging should be enabled                                                           |
-| AWS NIS 6.12 | S3 bucket object-level logging for read events should be enabled                                         |
-| AWS NIS 6.13 | S3 bucket object-level logging for write events should be enabled                                        |
+| Rule         | Description                                                                                                                        |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------------------- |
+| AWS NIS 1.1  | IAM role trust policies should not allow all principals to assume the role                                                         |
```

Comment:
I think these rules should start with the AWS NIST prefix

Reply:
yes, I think I need glasses XD. Thanks!